Task-5/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"FrontEndToDB/myconn"
+	myQueries "FrontEndToDB/mydb"
+	"fmt"
+	"html/template"
+
+	"net/http"
+)
+
+func main() {
+
+	// Making connection to postgres server
+	db := myconn.ConnectToDB(5555, "postgres", "1234", "postgres")
+	defer db.Close()
+
+	// Open static Path for ./HTML Form and Table files
+	fileServer := http.FileServer(http.Dir("./HTML Form and Table"))
+
+	// Handle '/' Route for Static Files
+	http.HandleFunc("/", fileServer.ServeHTTP)
+
+	// Insert Route handling for Customer database fetching value from FORM
+	http.HandleFunc("/pushData", func(w http.ResponseWriter, r *http.Request) {
+		err := r.ParseForm()
+		myconn.CheckErr(err)
+		// If Email is Empty send BadRequest
+		if r.Form.Get("email") == "" {
+			w.WriteHeader(http.StatusBadRequest)
+			w.Write([]byte("<h1>Please Enter valid Data </h1>"))
+			return
+		}
+		// Insert Data into postgres database
+		e := myQueries.InsertQuery(db, r.Form.Get("fname"), r.Form.Get("lname"), r.Form.Get("dob"), r.Form.Get("email"), r.Form.Get("number"))
```

Comment:
you can improve code to below two way use any below method.

1. pass whole request don't pass different data.
2. you can create structure and map those request data to structure and pass structure.

Reply:
@Tridipchavda